package.json

```diff
@@ -42,35 +41,31 @@
   "devDependencies": {
     "@salesforce/cli-plugins-testkit": "^2.5.2",
     "@salesforce/dev-config": "^3.0.1",
-    "@salesforce/dev-scripts": "^2.0.3",
+    "@salesforce/dev-scripts": "^3.1.0",
     "@salesforce/prettier-config": "^0.0.2",
     "@salesforce/ts-sinon": "^1.4.0",
     "@types/archiver": "^5.3.1",
     "@types/deep-equal-in-any-order": "^1.0.1",
     "@types/mime": "2.0.3",
-    "@types/mkdirp": "0.5.2",
     "@types/proxy-from-env": "^1.0.1",
     "@types/shelljs": "^0.8.11",
     "@types/unzipper": "^0.10.5",
-    "@typescript-eslint/eslint-plugin": "^4.33.0",
-    "@typescript-eslint/parser": "^4.33.0",
+    "@typescript-eslint/eslint-plugin": "^5.33.0",
+    "@typescript-eslint/parser": "^5.33.0",
     "chai": "^4.3.6",
     "commitizen": "^3.1.2",
-    "cz-conventional-changelog": "^2.1.0",
     "deep-equal-in-any-order": "^1.1.19",
     "deepmerge": "^4.2.2",
-    "eslint": "^7.32.0",
-    "eslint-config-prettier": "^6.15.0",
-    "eslint-config-salesforce": "^0.1.6",
+    "eslint": "^8.21.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-config-salesforce": "^1.1.0",
     "eslint-config-salesforce-license": "^0.1.6",
-    "eslint-config-salesforce-typescript": "^0.2.8",
+    "eslint-config-salesforce-typescript": "^1.1.1",
     "eslint-plugin-header": "^3.1.1",
     "eslint-plugin-import": "^2.26.0",
-    "eslint-plugin-jsdoc": "^35.5.1",
-    "eslint-plugin-prettier": "^4.0.0",
+    "eslint-plugin-jsdoc": "^39.3.6",
     "husky": "^7.0.4",
-    "jsforce": "2.0.0-beta.18",
-    "lint-staged": "^10.5.4",
+    "jsforce": "^2.0.0-beta.19",
```

Comment:
I've seen this just say `beta` in other repos.  should it be just beta or a specific beta version like above?

Reply:
no, because that doesn't `^` properly.  This causes the lockfiles to update better without having to manually change them all.